test/spec/modules/33acrossAnalyticsAdapter_spec.js

```diff
@@ -471,6 +481,81 @@ describe('33acrossAnalyticsAdapter:', function () {
         });
       });
     });
+
+    context('when the transaction manager has open transactions', function () {
+      it('reports those transactions as pending', function () {
+        this.enableAnalytics();
+
+        const { prebid: [auction] } = getMockEvents();
+        events.emit(EVENTS.AUCTION_INIT, auction.AUCTION_INIT);
+        events.emit(EVENTS.BID_REQUESTED, auction.BID_REQUESTED[0]);
+
+        const manager = locals.transactionManagers[auction.AUCTION_INIT.auctionId];
+        assert.equal(manager.status().pending.length, auction.BID_REQUESTED[0].bids.length);
+      });
+
+      context('and a single bidWon event has triggered', function () {
+        it('completes the transaction', function () {
+          this.enableAnalytics();
+
+          const { prebid: [auction] } = getMockEvents();
+          events.emit(EVENTS.AUCTION_INIT, auction.AUCTION_INIT);
+          events.emit(EVENTS.BID_REQUESTED, auction.BID_REQUESTED[0]);
+          events.emit(EVENTS.BID_WON, auction.BID_WON[0]);
+
+          const manager = locals.transactionManagers[auction.AUCTION_INIT.auctionId];
+          assert.deepEqual({
+            completed: manager.status().completed.length,
+            pending: manager.status().pending.length
+          }, {
+            completed: 1,
+            pending: auction.BID_REQUESTED[0].bids.length - 1
+          });
+        });
+      });
+
+      context('and a single slotRenderEnded event has triggered', function () {
+        it('does NOT complete the transaction if the GAM timeout has not elapsed', function () {
```

Comment:
We usually don't use conditions in the unit test titles.. maybe use another context wrapper
```
 context('and a single slotRenderEnded event has triggered', function () {
   context('and the Google Ad Manager timeout has not elapsed', function() {
     it('does NOT complete the transaction')
   })
```

Please change other unit tests that also have a condition.